jar-infer/test-java-lib-jarinfer/build.gradle

```diff
@@ -23,7 +23,7 @@ evaluationDependsOn(":jar-infer:jar-infer-cli")
 sourceCompatibility = "1.8"
 targetCompatibility = "1.8"
 
-def astubxPath = "META-INF/nullaway/jarinfer.astubx"
+def astubxPath = "testastubx/jarinfer.astubx"
```

Comment:
I don't think there are any real rules as to where you can and can't put the `astubx` files in a jar.  AFAIK resources packaged in a jar can end up in any directory (except I don't think they would end up under `META-INF` since that folder is special).